crates/re_viewer/src/ui/selection_panel.rs

```diff
@@ -308,17 +310,28 @@ fn what_is_selected_ui(
         Item::SpaceView(space_view_id) => {
             if let Some(space_view) = viewport.space_view(space_view_id) {
                 let space_view_class = space_view.class(ctx.space_view_class_registry);
-                item_title_ui(
-                    ctx.re_ui,
-                    ui,
-                    &space_view.display_name,
-                    Some(space_view_class.icon()),
-                    &format!(
+
+                let hover_text = if let Some(display_name) = space_view.display_name.as_ref() {
+                    format!(
                         "Space View {:?} of type {}",
-                        space_view.display_name,
-                        space_view_class.display_name(),
-                    ),
-                );
+                        display_name,
+                        space_view_class.display_name()
+                    )
+                } else {
+                    format!(
+                        "Unnamed Space View of type {}",
+                        space_view_class.display_name()
+                    )
+                };
+
+                let (label, named) = space_view.display_name_or_default();
+                ListItem::new(ctx.re_ui, label)
+                    .unnamed_style(!named)
```

Comment:
`ListItem::new` already takes an `WidgetText` argument. Doesn't it make more sense to separate the formatting of the label from the rest of the `ListItem` logicl?

Reply:
Excellent point. All this styling stuff cramed into `ListWidget` was really annoying me. I'll do that in a separate PR though: https://github.com/rerun-io/rerun/issues/4802